stable/prometheus/templates/alertmanager-statefulset.yaml

```diff
@@ -32,6 +32,9 @@ spec:
 {{- if .Values.alertmanager.priorityClassName }}
       priorityClassName: "{{ .Values.alertmanager.priorityClassName }}"
 {{- end }}
+      securityContext:
```

Comment:
It's probably better to move all the securityContext definitions to values.yaml and use toYaml in case it causes difficulties in some clusters. Also people might want to customize other options like capabilities, etc.

Reply:
one securityContext-definition per component? (alertmanager, prometheus, kube-state-metrics, etc.)

but with these default-values?
```
      securityContext:
        runAsUser: 65534
        fsGroup: 65534
```

Reply:
Yes please. But add `runAsGroup: 65534` and `runAsNonRoot: true`.

Reply:
didn't see, that each component already have the `securityContext` defined on a container-level.

i updated the `values.yaml` so if this PR got merged, the default behavior will enable all defined `securityContext` flags.